emscripten.py

```diff
@@ -1797,7 +1784,7 @@ def create_fp_accessors(metadata):
   var func = Module['%(mangled)s'];
   if (!func)
     func = %(mangled)s;
-  var fp = addFunction(func, '%(sig)s');
+  var fp = addFunctionWasm(func, '%(sig)s');
```

Comment:
There are two changes like this in this PR. Perhaps they have an effect? In fastcomp, `addFunction` will not call `addFunctionWasm` necessarily, it depends on `EMULATED_FUNCTION_POINTERS`.

If that's not it, then one way to debug the OOM issue is to add some logging of each call to `addFunctionWasm`, and to view that logging before and after this PR.